restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/ConferencesEndpoint.java

```diff
@@ -194,4 +207,77 @@ protected Response getConferences(final String accountSid, UriInfo info, MediaTy
         }
     }
 
+    protected Response updateConference(String accountSid, String sid, MultivaluedMap<String, String> data,
+            MediaType responseType) {
+        logger.info(String.format("updateConference accountsid: %s conferenceSid: %s", accountSid, sid));
+        Account account = daoManager.getAccountsDao().getAccount(accountSid);
+        try {
+            secure(account, "RestComm:Modify:Conferences");
+        } catch (final AuthorizationException exception) {
+            return status(UNAUTHORIZED).build();
+        }
+        final ConferenceDetailRecordsDao dao = daoManager.getConferenceDetailRecordsDao();
+        ConferenceDetailRecord cdr = dao.getConferenceDetailRecord(new Sid(sid));
+        if (cdr == null) {
+            return status(NOT_FOUND).build();
+        } else {
+            try {
+                secure(account, cdr.getAccountSid(), SecuredType.SECURED_STANDARD);
+            } catch (final AuthorizationException exception) {
+                return status(UNAUTHORIZED).build();
+            }
+
+            final String status = data.getFirst("Status");
+
+            if(status != null){
+                if(status.equalsIgnoreCase("completed")){
+                    kickoutAllActiveParticipants(cdr);
+                    //get updated conference record
+                    cdr = dao.getConferenceDetailRecord(new Sid(sid));
+                }
+            }
+            if (APPLICATION_XML_TYPE == responseType) {
+                final RestCommResponse response = new RestCommResponse(cdr);
+                return ok(xstream.toXML(response), APPLICATION_XML).build();
+            } else if (APPLICATION_JSON_TYPE == responseType) {
+                return ok(gson.toJson(cdr), APPLICATION_JSON).build();
+            } else {
+                return null;
+            }
+        }
+    }
+
+    private void kickoutAllActiveParticipants(ConferenceDetailRecord conferenceDetailRecord){
+        List<CallDetailRecord> callDetailRecords = daoManager.getCallDetailRecordsDao().getRunningCallDetailRecordsByConferenceSid(conferenceDetailRecord.getSid());
+
+        if(callDetailRecords == null || callDetailRecords.isEmpty()){
```

Comment:
@maria-farooq also here I think we need a response to tell user that there were no participants

Reply:
@gvagenas I dont think it need other than a suceesful response, user will get conference resource in response.
this case here is just to make sure in a race case when participants leave themselves before system kick them out

Reply:
main issue with any single response is that this would require to wait for result on async actions started from here, and which may take time (LVM remote invocation). The response should be succesful if the action of terminating the conf may be initiated.

I think any client can check the RESTAPI later to see how many participants are left at any moment...